src/actions.js

```diff
@@ -211,6 +211,7 @@ export function formatClaimGQL(mm, claim) {
     feedbackStatus: ${mm.getRef("claim.CreateClaim.feedbackStatus")}
     reviewStatus: ${mm.getRef("claim.CreateClaim.reviewStatus")}
     dateClaimed: "${claim.dateClaimed}"
+    ${claim.visitType==='R' ? `referFromId` : `referToId`}: ${decodeId(claim.referHF.id)}
```

Comment:
I will suggest to create a constant which will contain 'R' string. Moreover, use white spaces before/after equal signs. It will improve the clarity of the code.